server/admin_management/api/app/jwt_validation.py

```diff
@@ -211,3 +210,9 @@ def get_request_cognito_user_id(claims: dict = Depends(authorize)):
     cognito_username = claims[COGNITO_USERNAME_KEY]
     LOGGER.debug(f"Current requester's cognito_username for API: {cognito_username}")
     return cognito_username
+
+
+def get_request_cognito_user_id_without_access_check(claims: dict = Depends(validate_token)):
+    cognito_username = claims[COGNITO_USERNAME_KEY]
+    LOGGER.debug(f"Current requester's cognito_username for API: {cognito_username}")
+    return cognito_username
```

Comment:
get user's cognito username without checking the access role in the token, so this method depends on method validate_token, instead of method authorize